### PR TITLE
 Convert script to and from an address. Implements #35 

### PR DESCRIPTION
**Pull request by ghost**
Depends on #30.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,10 @@
     "url": "https://github.com/JBaczuk/bscript-parser/issues"
   },
   "homepage": "https://github.com/JBaczuk/bscript-parser#readme",
-  "dependencies": {},
+  "dependencies": {
+    "bech32": "^1.1.3",
```
Comment (JBaczuk) [inline package.json]: lock these dependencies

File: src/bscript.js [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+}
+
+const ADDRESS_SCRIPT_TYPES = [
+  'p2pkh',
+  'p2sh',
+  'p2wsh',
+  'p2wpkh'
+]
+
+const PUBKEY_HASH = 0x00
```
Comment (JBaczuk) [inline src/bscript.js]: I know this is for bitcoin, but it would be good to parameterize these to make this library available to other similar coins.

**Review by JBaczuk — CHANGES_REQUESTED** [review]
Looks good, just a few minor things.

File: src/bscript.js (hunk 2 above)
Comment (wardlem) [reply src/bscript.js]: @JBaczuk These constants are just defaults.  They can be passed in as the `pubKeyHash`, `scriptHash`, and `bech32` options.

File: package.json (hunk 1 above)
Comment (wardlem) [reply package.json]: @JBaczuk This is done.  I also rebased to resolve the conflict in the package.json file.

**Review by JBaczuk — APPROVED** [review]

**Pull request merged**